### PR TITLE
Don't launch Pecan if the table is not georeferenced

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds a new check in Pecan to disable the analysis if the table is not georeferenced.

Original ticket: #5005

### CODE REVIEW

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_card.js [hunk 1]
```diff
@@ -67,45 +67,6 @@ module.exports = cdb.core.View.extend({
     this.trigger("click", this.model, this);
   },
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_card.js]: I found this old code that is not used here (it was moved to another class in a code refactor)

**Comment by javierarce** [comment]
@javisantana: can you have a look, please?

**Comment by javierarce** [comment]
retest this please

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js [hunk 2]
```diff
@@ -40,7 +42,7 @@ module.exports = cdb.core.View.extend({
     var col_count = _(this.query_schema).size();
     var hasColumns = col_count > 0 && col_count < this._MAX_COLS;
 
-    if (hasGeometries && hasRows && hasColumns) {
+    if (isGeoreferenced && hasGeometries && hasRows && hasColumns) {
       return true;
     } else {
       return false;
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js]: `return isGeoreferenced && hasGeometries && hasRows && hasColumns` perhaps maybe?

Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js]: Yeah, I'll change it.

**Comment by javisantana** [comment]
looks good

**Comment by javierarce** [comment]
retest this please

**Pull request merged**